roles/openshift_node/tasks/upgrade/config_changes.yml

```diff
@@ -1,7 +1,9 @@
 ---
-- name: Update systemd units
-  import_tasks: ../systemd_units.yml
-  when: openshift_is_containerized | bool
+- name: Ensure the node static pod directory exists
```

Comment:
It would be nice if we could break this file apart into component areas.  It's just really large as is and it looks like we can probably optimize the ordering of some things to group them into logical units.

Reply:
Yes

Reply:
Slightly better factored now